web/react/components/edit_post_modal.jsx

```diff
@@ -26,79 +34,102 @@ module.exports = React.createClass({
         updatedPost.channel_id = this.state.channel_id;
 
         Client.updatePost(updatedPost,
-            function(data) {
+            function success() {
                 AsyncClient.getPosts(this.state.channel_id);
                 window.scrollTo(0, 0);
             }.bind(this),
-            function(err) {
-                AsyncClient.dispatchError(err, "updatePost");
-            }.bind(this)
+            function error(err) {
+                AsyncClient.dispatchError(err, 'updatePost');
+            }
         );
 
-        $("#edit_post").modal('hide');
+        $('#edit_post').modal('hide');
         $(this.state.refocusId).focus();
-    },
-    handleEditInput: function(editMessage) {
+    }
+    handleEditInput(editMessage) {
         this.setState({editText: editMessage});
-    },
-    handleEditKeyPress: function(e) {
-        if (e.which == 13 && !e.shiftKey && !e.altKey) {
+    }
+    handleEditKeyPress(e) {
+        if (e.which === 13 && !e.shiftKey && !e.altKey) {
             e.preventDefault();
             this.refs.editbox.getDOMNode().blur();
```

Comment:
There's an instance of getDOMNode here as well as on the following:
components/file_attachment.jsx:46
components/file_attachment.jsx:84
components/mention_list.jsx:68
a bunch in components/new_channel.jsx
components/team_signup_display_name_page.jsx:24

Same as my other comment, there might be some I missed that I can't see on this page.